### PR TITLE
[WOR-389] Add last_modified to Billing Profile Manager (BPM)

### PR DESCRIPTION
**Pull request by yunip20**
Adds last_modified to billing profile manager

https://broadworkbench.atlassian.net/browse/WOR-389

### CODE REVIEW

**Comment by aherbst-broad** [comment]
Thanks for doing this, I'll take a look! Somewhat related: I typically include the ticket in the branch name for easy reference (i.e., if I'm working on it, put it aside and come.back to it a day later). This makes picking out the correct branch to switch back to much easier (something like yp-wor-389-blah-blah). No need to change the name of this branch though.

**Comment by yunip20** [comment]
I understand, thank you!

File: service/src/main/resources/api/service_openapi.yaml [hunk 1]
```diff
@@ -318,6 +318,9 @@ components:
         createdDate:
           type: string
           description: Date the profile was created
+        lastModified:
+          type: string
+          description: Date the profile was last modified
```
Comment (aherbst-broad) [inline service/src/main/resources/api/service_openapi.yaml:323]: The documentation here isn't quite correct, this is the timestamp in UTC the profile was last modifed, not the date.

File: service/src/main/resources/db/changesets/20220713_billing_profiles_last_modified.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      changes:
+        - addColumn:
+            tableName: billing_profile
+            columns:
+              - column:
+                  name: last_modified
+                  type: timestamptz
+                  defaultValueComputed: now()
+                  remarks: |
+                    Date the profile was last modified.
```
Comment (aherbst-broad) [inline service/src/main/resources/db/changesets/20220713_billing_profiles_last_modified.yaml:14]: Nitpick: add a line break here

**Review by aherbst-broad — COMMENTED** [review]
Looking good, had a few suggestions. I think there are also some linting issues judging by the CI failure.

File: service/src/main/resources/api/service_openapi.yaml (hunk 1 above)
Comment (aherbst-broad) [reply service/src/main/resources/api/service_openapi.yaml:323]: @yunip20 can you update this documentation as well?

**Review by aherbst-broad — APPROVED** [review]

**Pull request merged**